content_blocker_update.sh

```diff
@@ -1,20 +0,0 @@
-#!/bin/sh
-set -e
-
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-# Version 107.0 hash
-SHAVAR_COMMIT_HASH="91cf7dd142fc69aabe334a1a6e0091a1db228203"
-
-# Install Node.js dependencies and build user scripts
-npm install
-npm run build
-
-# Clone shavar prod list
-rm -rf shavar-prod-lists && git clone https://github.com/mozilla-services/shavar-prod-lists.git && git -C shavar-prod-lists checkout $SHAVAR_COMMIT_HASH
-
-cd BrowserKit
-swift run || true
-swift run
```

Comment:
Note: I believe removal of this also means we can clean up some of the related code in BrowserKit (e.g. `ContentBlockerGenerator`)?

Reply:
Yes. I converted it to a simpler python script. My initial plan was to use it as is and have docker multistage build ( like I did in [this tmp repo](https://github.com/issammani/tracking-protection-lists-webkit/blob/7510086b054a3129a5aebbf708047a1acaa571b0/Dockerfile#L27-L38 ) ) to build a binary from swift and use it, but that overcomplicates the build process. We don't need that code anymore.

Reply:
Ok thanks, I might create a follow-up task for that cleanup and just put up a quick separate PR to remove it.